### PR TITLE
refactor(UtilService): Move rgbToHex()

### LINKED ISSUE #993: refactor(UtilService): Move rgbToHex()
This function doesn't need to be in UtilService. Move to its own file.

### PR DESCRIPTION
**Pull request by breity**
## Changes
- Move rgbToHex() to it's own file.
- Add check for proper rgb string format.

Closes #993.

## Test
- Ensure that milestone report graphs display color correctly.
- Ensure the milestone report graphs with multiple locations display different shades of set color for the different location data points.
- Ensure that summary graphs (student and teacher) show proper color and shading for different scores.

### CODE REVIEW

File: src/assets/wise5/common/color/color.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { rgbToHex } from './color';
+
+describe('canvas', () => {
```
Comment (hirokiterashima) [inline src/assets/wise5/common/color/color.spec.ts]: Remove outer describe block?

File: src/assets/wise5/common/color/color.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export function rgbToHex(color: string, opacity?: any): string {
```
Comment (hirokiterashima) [inline src/assets/wise5/common/color/color.ts]: is opacity a number type?

**Review by hirokiterashima — APPROVED** [review]
Looks good. I added minor code improvement suggestions/questions, but otherwise good to merge.

File: src/assets/wise5/common/color/color.ts (hunk 2 above)
Comment (breity) [reply src/assets/wise5/common/color/color.ts]: Yep, opacity can be typed to a number. I added that and also added a default value. Those changes let us remove the `parseFloat(opacity || 1)` and just use the opacity param in the r, g, b calculations.

**Pull request merged**